spec/factories/rules.rb

```diff
@@ -95,4 +111,197 @@
     name 'Minimum occupancy'
     verb 'have'
   end
+  factory :interested_in_cambridge, class: 'Rules::InterestedInNeighborhood' do
```

Comment:
Shouldn't this be `interested_in_neighborhood`? The requirement would need a "variable" to specify the specific neighborhood.